tests/test_issues.py

```diff
@@ -257,45 +257,6 @@ def test_issue_359():
         video.write_gif(filename=os.path.join(TMP_DIR, "issue_359.gif"), tempfiles=True)
 
 
-# TODO: Debug matplotlib failures following successful travis builds.
-# def test_issue_368():
-#     # Travis/3.5 fails.
-#     if PYTHON_VERSION == '3.5' and TRAVIS:
-#        return
-#
-#     import numpy as np
-#     import matplotlib.pyplot as plt
-#     from sklearn import svm
-#     from sklearn.datasets import make_moons
-#     from moviepy.video.io.bindings import mplfig_to_npimage
-#
-#     X, Y = make_moons(50, noise=0.1, random_state=2) # semi-random data
-#
-#     fig, ax = plt.subplots(1, figsize=(4, 4), facecolor=(1,1,1))
-#     fig.subplots_adjust(left=0, right=1, bottom=0)
-#     xx, yy = np.meshgrid(np.linspace(-2,3,500), np.linspace(-1,2,500))
-#
-#     def make_frame(t):
-#         ax.clear()
-#         ax.axis('off')
-#         ax.set_title("SVC classification", font_size=16)
-#
-#         classifier = svm.SVC(gamma=2, C=1)
-#         # the varying weights make the points appear one after the other
-#         weights = np.minimum(1, np.maximum(0, t**2+10-np.arange(50)))
-#         classifier.fit(X, Y, sample_weight=weights)
-#         Z = classifier.decision_function(np.c_[xx.ravel(), yy.ravel()])
-#         Z = Z.reshape(xx.shape)
-#         ax.contourf(xx, yy, Z, cmap=plt.cm.bone, alpha=0.8,
-#                     vmin=-2.5, vmax=2.5, levels=np.linspace(-2,2,20))
-#         ax.scatter(X[:,0], X[:,1], c=Y, s=50*weights, cmap=plt.cm.bone)
-#
-#         return mplfig_to_npimage(fig)
-#
-#     animation = VideoClip(make_frame, duration=2)
-#     animation.write_gif(os.path.join(TMP_DIR, "svm.gif"), fps=20)
-
-
```

Comment:
I think that this test should stay. It was initially only skipped on Travis because it didn't work there. Then it was completely commented out (presumably because it didn't work in other places).

So you can remove the Travis checks from it, and leave it commented out if it still doesn't work, but I don't see why it should be removed entirely.

Reply:
Okay, I will try to make it work :+1:

Reply:
Well I wouldn't worry about that too much, I just think that it shouldn't be deleted :)
The matplotlib parts of moviepy are very far of the main use case of the library and probably don't work very well thees days anyway.

Reply:
The test is working now. I've added optional requirements installation to Windows and MacOSX builds, to install matplotlib. Also, used the backend "agg" for matplotlib in the test.